### PR TITLE
Add TotalAccountBalanceExample

### PR DESCRIPTION
**Pull request by puzatin**

### CODE REVIEW

**Comment by joaopsilva** [comment]
Thanks for the contribution, please see my comments.

**Comment by puzatin** [comment]
I understand you correctly that it was necessary to add comments?

File: src/test/java/com/binance/api/examples/TotalAccountBalanceExample.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    public static void main(String[] args) {
+        BinanceApiClientFactory factory = BinanceApiClientFactory.newInstance("YOUR_API_KEY", "YOUR_SECRET");
+        BinanceApiRestClient client = factory.newRestClient();
+
+        // Get account balances
+        Account account = client.getAccount(60_000L, System.currentTimeMillis());
+
+        // Get total account balance in BTC (spot only)
+        TotalAccountBalanceExample accountBalance = new TotalAccountBalanceExample();
```
[the remark this answers is not in the document]
Comment (puzatin) [reply src/test/java/com/binance/api/examples/TotalAccountBalanceExample.java]: i added it, this is the class you are commenting on

File: src/main/java/com/binance/api/client/constant/Util.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import java.util.List;
+
+public final class Util {
+
+    public static final List<String> FIAT_CURRENCY = Collections.unmodifiableList(Arrays.asList("USDT", "BUSD", "PAX", "TUSD", "USDC", "NGN", "RUB", "USDS, TRY"));
+
+    public static final String BTC_TICKER = "BTC";
+
+    private Util() {
+        throw new java.lang.UnsupportedOperationException("This is a utility class and cannot be instantiated");
```
[the remark this answers is not in the document]
Comment (puzatin) [reply src/main/java/com/binance/api/client/constant/Util.java]: done

**Comment by joaopsilva** [comment]
Thanks.

**Pull request merged**